auto-value-parcel/src/main/java/com/ryanharter/auto/value/parcel/Parcelables.java

```diff
@@ -49,11 +50,15 @@
   static final TypeName SIZEF = ClassName.get("android.util", "SizeF");
   static final TypeName TEXTUTILS = ClassName.get("android.text", "TextUtils");
   static final TypeName ENUM = ClassName.get(Enum.class);
+  static final TypeName IMMUTABLE_LIST = ClassName.get("com.google.common.collect", "ImmutableList");
```

Comment:
AutoValue itself supports any kinds of immutables, as long as they implement the parent interface and contain a static `copyOf()` method. I think we should match that, as not eceryone uses guava directly